h/renderers.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pyramid.renderers
+
+
+json_sorted = pyramid.renderers.JSON(sort_keys=True)
```

Comment:
This threw me off for a second, the variable is called `json_sorted` and further down is used as the value for the argument `factory`. I checked out the Pyramid source code and they do: `json_renderer_factory = JSON()`.
Would you think it's better to name this `json_sorted_factory`?